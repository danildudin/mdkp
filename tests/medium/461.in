28 5
868 171 766 806 428 640 618 852 463 616 946 546 656 6 977 960 875 415 544 950 964 911 264 714 28 485 719 151 
28 723 530 684 136 370 268 561 755 894 987 996 431 137 375 510 504 364 625 742 741 533 430 175 787 405 860 160 9977
762 408 49 735 990 257 636 413 686 451 323 360 712 793 64 553 740 430 934 999 523 400 706 790 484 377 571 253 5009
740 731 834 538 683 678 861 381 43 423 111 162 757 247 493 417 731 128 24 486 656 210 340 463 716 341 69 243 7153
641 154 320 390 852 229 283 287 124 611 143 129 402 430 491 261 385 992 390 481 671 17 544 610 274 419 95 463 7575
227 742 226 51 9 861 465 178 576 864 448 642 203 754 254 665 629 339 994 137 608 212 131 997 639 940 449 852 8851
