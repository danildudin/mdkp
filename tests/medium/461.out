10000
13
2 5 6 7 9 10 11 14 15 20 21 25 26 
